src/spryker/application/skeleton/config/install/docker.yml.twig

```diff
@@ -91,17 +91,24 @@ sections:
     cache:
         router-cache-warmup-yves:
             command: "vendor/bin/yves router:cache:warm-up"
+            stores: true
 
         router-cache-warmup-backoffice:
             command: 'vendor/bin/console router:cache:warm-up:backoffice'
+            stores: true
 
         router-cache-warmup-backend-gateway:
             command: 'vendor/bin/console router:cache:warm-up:backend-gateway'
+            stores: true
 
         twig-cache-warmup:
             command: "vendor/bin/console twig:cache:warmer"
             stores: true
 
+        translator-generate-cache:
+            command: "vendor/bin/console translator:generate-cache"
+            stores: true
```

Comment:
This didn't have `stores: true` before. Does it need to now? https://github.com/inviqa/harness-base-php/blob/1.4.x/src/spryker/application/skeleton/config/install/docker.yml.twig#L111-L112

Reply:
main way to check is to check the folder it writes to. If it has per store folders then it probably needs to be true. If not, then it may be running the same thing X*X times for X stores

Reply:
I believe it does according to https://github.com/spryker/translator/blob/master/src/Spryker/Zed/Translator/TranslatorConfig.php#L100

and in our test harness I have
`root@27f7128bfaa7:/app# ls ./src/Generated/Zed/Translator/codeBucketInviqa`

`root@27f7128bfaa7:/app# ls ./src/Generated/Zed/Translator/codeBucket/Inviqa/Pyz`

I assume that is our dummy store?

Reply:
no it'd look more like a locale. Pyz is a spryker class override method

Reply:
I wouldn't expect it to need multistore for this but if in doubt check to see what the original demoshop is doing: https://github.com/spryker-shop/b2c-demo-shop/blob/master/config/install/docker.yml#L40

In this case looks like it's ok without it.